test/e2e/clusterctl_upgrade_test.go

```diff
@@ -62,24 +62,24 @@ var _ = Describe("When testing clusterctl upgrades (v1.4=>current)", func() {
 			ArtifactFolder:         artifactFolder,
 			SkipCleanup:            skipCleanup,
 			InfrastructureProvider: pointer.String("docker"),
-			InitWithBinary:         "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.4.5/clusterctl-{OS}-{ARCH}",
+			InitWithBinary:         "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.5.4/clusterctl-{OS}-{ARCH}",
 			// We have to pin the providers because with `InitWithProvidersContract` the test would
-			// use the latest version for the contract (which is v1.5.X for v1beta1).
-			InitWithCoreProvider:            "cluster-api:v1.4.5",
-			InitWithBootstrapProviders:      []string{"kubeadm:v1.4.5"},
-			InitWithControlPlaneProviders:   []string{"kubeadm:v1.4.5"},
-			InitWithInfrastructureProviders: []string{"docker:v1.4.5"},
+			// use the latest version for the contract (which is v1.6.X for v1beta1).
+			InitWithCoreProvider:            "cluster-api:v1.5.4",
+			InitWithBootstrapProviders:      []string{"kubeadm:v1.5.4"},
+			InitWithControlPlaneProviders:   []string{"kubeadm:v1.5.4"},
+			InitWithInfrastructureProviders: []string{"docker:v1.5.4"},
 			InitWithProvidersContract:       "v1beta1",
-			// NOTE: If this version is changed here the image and SHA must also be updated in all DockerMachineTemplates in `test/e2e/data/infrastructure-docker/v1.4/bases.
```

Comment:
is this still true? looks like the hardcoded SHA refs were removed from the templates in the last cycle: https://github.com/kubernetes-sigs/cluster-api/pull/9097/files#diff-bad8038a4f09fe4c891e6c9021ec75602c324487810f68cce202b71967ac171aL11-L12

Reply:
You're right. We should drop this comment here and in other places below (this only still applies to 1.0)